db/models.py

```diff
@@ -37,7 +37,7 @@ class CircuitScheduleDoc(BaseModel):
 
 class TAGDoc(BaseModel):
     """TAG model"""
-    tag_type: int
+    tag_type: str
```

Comment:
We'll also need a migration script since EVCs that were created otherwise when loading an EVC they'll result in `TAGType` enum value error.

Reply:
Script added